rotkehlchen/chain/evm/decoding/stakedao/utils.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from collections import defaultdict
+from typing import Final
+
+from rotkehlchen.errors.misc import RemoteError, UnableToDecryptRemoteData
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.globaldb.cache import globaldb_set_general_cache_values
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import deserialize_evm_address
+from rotkehlchen.types import CacheType, ChainID
+from rotkehlchen.utils.network import request_get_dict
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+STAKEDAO_API: Final = 'https://api.stakedao.org/api'
+# This is gotten from:
+# https://github.com/stake-dao/api?tab=readme-ov-file#strategies-data
+SUPPORTED_STAKEDAO_STRATEGIES: Final = {'curve', 'balancer', 'pendle', 'pancakeswap', 'angle'}
+
+
+def query_stakedao_gauges() -> None:
+    """Query StakeDAO lockers and strategies gauges from the API and cache them."""
```

Comment:
```suggestion
    """Query StakeDAO guages for lockers and strategies from the API and cache them."""
```
I think this is better. When I read the original I thought it meant that it was getting `lockers` and `strategy guages`.